pyqwikswitch/qwikswitch.py

```diff
@@ -108,7 +109,7 @@ class QSDev():
 
     def __attrs_post_init__(self):
         """Init."""
-        _types = {'rel': QSType.relay, 'dim': QSType.dimmer}
+        _types = {'rel': QSType.relay, 'dim': QSType.dimmer, 'hum': QSType.humidity_temperature}
```

Comment:
Can you please wrap this as

```
       _types = {'rel': QSType.relay, 'dim': QSType.dimmer,
                 'hum': QSType.humidity_temperature}
```

Since it's mostly targeted at Home Assistant I try to follow all their coding [guidelines](https://developers.home-assistant.io/docs/en/development_guidelines.html) (80 char wrap) pylint & flake8

Reply:
Setup your environment with:

```
pip3 install flake8 flake8-docstrings pylint
flake8
pylint
```